Makefile

```
@@ -1,11 +1,24 @@
-.PHONY: testshort
-testshort:
+help: ## Print usage
+	@sed -r '/^(\w+-*\w+):[^#]*##/!d;s/^([^:]+):[^#]*##\s*(.*)/\x1b[36m\1\t:\x1b[m \2/g' ${MAKEFILE_LIST}
```

Missing .PHONY on help

```suggestion
	@sed -r '/^(\w+-*\w+):[^#]*##/!d;s/^([^:]+):[^#]*##\s*(.*)/\x1b[36m\1\t:\x1b[m \2/g' ${MAKEFILE_LIST}
.PHONY: help
```